src/explore-education-statistics-frontend/src/modules/find-statistics/components/MobileFilters.tsx

```diff
@@ -24,24 +23,36 @@ const MobileFilters = ({
   themes,
   totalResults,
   onChange,
-  onClose,
 }: Props) => {
+  const mobileFilterButtonRef = useRef<HTMLButtonElement>(null);
```

Comment:
What's this for?

Reply:
It was left over from the previous version of the modal, it was used to refocus the button when the modal was closed but it's not needed anymore so I've removed it.